packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/saml2assert/flow-diagram.md

```diff
@@ -1,32 +1,23 @@
 ### SAML 2.0 Assertion flow
 
-<div class="three-quarter">
+<div class="full">
```

Comment:
these are all too big now - I think three-quarter is better. They're giant when I build this branch locally.